linenoise.hpp

```diff
@@ -1573,20 +1641,29 @@ inline int unicodeReadUTF8Char(int fd, char* buf, int* cp)
 /* ======================= Low level terminal handling ====================== */
 
 /* Set if to use or not the multi line mode. */
-inline void SetMultiLine(bool ml) {
+void linenoiseState::SetMultiLine(bool ml) {
     mlmode = ml;
 }
 
 /* Return true if the terminal name is in the list of terminals we know are
  * not able to understand basic escape sequences. */
-inline bool isUnsupportedTerm(void) {
+static bool isUnsupportedTerm(void) {
```

Comment:
Is there any reason to change it to `static` from `inline`?

Reply:
Not really - I guess I'm just more used to static.  I'll switch it back.